src/test/java/com/ays/institution/model/entity/InstitutionEntityBuilder.java

```diff
@@ -21,6 +21,11 @@ public InstitutionEntityBuilder withId(String id) {
         return this;
     }
 
+    public InstitutionEntityBuilder withName(String name) {
+        data.setName(name);
+        return this;
+    }
```

Comment:
Buraya neden ihtiyaç duyduk?

Reply:
InstitutionServiceImplTest'deki whenActiveInstitutionsExist_thenReturnInstitutions'da her bir institution için name vermiştim onun için kullandım.

Reply:
Selamlar, düzelttim.